modules/waypoint_tracking.py

```diff
@@ -34,19 +32,20 @@ def get_current_waypoint_info(
     # Get the current destination
     if current_waypoint < drone.commands.count:
         current_command = drone.commands[current_waypoint]
-        if current_command.command == dronekit.mavutil.mavlink.MAV_CMD_NAV_WAYPOINT:
+        destination_reached = drone.is_drone_destination_final_waypoint()
```

Comment:
this is the correct implementation! but the better practice would to store the two values returned in two separate variables instead of in a tuple
ex.
x, y = func()